src/content/general/overview/index.md

```diff
@@ -38,7 +41,7 @@ Your clusters comes out-of-the-box as follows:
 
 There are some things not included in the cluster as managed by us:
 
-- Additional user-space services like Dashboard, Monitoring, Logging, Registry, etc. are not installed (getting them running is [really easy]({{< relref "/app-platform/overview" >}}), though).
+- Additional user-space services like logging, container registry, etc. are not installed (getting them running is [really easy]({{< relref "/app-platform/overview" >}}), though).
 
```

Comment:
Removing the info that we don't offer dashboards

Reply:
I'm not certain I agree here - I wouldn't say our new dashboard work is user-space. i.e: you would still need to install dashboards and monitoring for any of your applications (to start with, at least)

Reply:
I see